.postcssrc.js

```diff
@@ -0,0 +1,24 @@
+
+module.exports = {
+	plugins: [
+		require('postcss-prefix-selector')({
+			prefix: '.enabled-vuetify-global-styling',
+			transform: function (prefix, selector, prefixedSelector) {
+				if ((selector.startsWith('.v-application') || selector.startsWith('.theme--light.v-application')) && !selector.startsWith('.v-application--')) {
+					return prefixedSelector;
+				}
+
+				if(selector.includes('html')) {
+					return prefixedSelector;
+				}
+
+				if(selector.includes('select') || selector.includes('textarea')) {
```

Comment:
these conditions are too general. It's very likely they will cause problems when we use these keywords in our selectores. Please convert them to somethinkg like:
* selector starts with X
* selector contains X preceded by space
* selector contains X preceded by comma